vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -736,19 +738,14 @@ def allowed_snapshots(_options = {})
   end
 
   def allowed_datastore_storage_controller(_options = {})
-    # TODO: NetAppFiler is an ActAsArModel and doesn't support arel syntax yet
-    NetAppFiler.all.collect(&:name).index_by { |n| n }
+    []
   end
+  deprecate :allowed_datastore_storage_controller
```

Comment:
@tenderlove Does this log to the Rails logger by default?  I guess it's fine to do that.  We typically log everything to $log but maybe it makes sense to just do it the "rails way" and avoid having to have our own deprecation logging code.  cc @Fryguy @matthewd 

See one example [here](https://github.com/jrafanie/manageiq/blob/a408eae34689348882640d162640f64adbd5a1c5/vmdb/app/models/relationship.rb#L188)